### PR TITLE
ISPN-9797 Ensure the server starts with the correct JDK modules if ne…

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-9797

### CODE REVIEW

File: server/integration/feature-pack/src/main/resources/content/bin/standalone.sh [hunk 1]
```diff
@@ -285,40 +265,38 @@ if [ "$PRESERVE_JAVA_OPTS" != "true" ]; then
...
+        if [ "x$NO_GC_LOG_ROTATE" = "x" ]; then
+            # backup prior gc logs
+            mv -f "$JBOSS_LOG_DIR/gc.log.0" "$JBOSS_LOG_DIR/backupgc.log.0" >/dev/null 2>&1
+            mv -f "$JBOSS_LOG_DIR/gc.log.1" "$JBOSS_LOG_DIR/backupgc.log.1" >/dev/null 2>&1
+            mv -f "$JBOSS_LOG_DIR/gc.log.2" "$JBOSS_LOG_DIR/backupgc.log.2" >/dev/null 2>&1
+            mv -f "$JBOSS_LOG_DIR/gc.log.3" "$JBOSS_LOG_DIR/backupgc.log.3" >/dev/null 2>&1
+            mv -f "$JBOSS_LOG_DIR/gc.log.4" "$JBOSS_LOG_DIR/backupgc.log.4" >/dev/null 2>&1
+            mv -f "$JBOSS_LOG_DIR"/gc.log.*.current "$JBOSS_LOG_DIR/backupgc.log.current" >/dev/null 2>&1
+            "$JAVA" $JVM_OPTVERSION -verbose:gc -Xloggc:"$JBOSS_LOG_DIR/gc.log" -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=3M -XX:-TraceClassUnloading -version >/dev/null 2>&1 && mkdir -p $JBOSS_LOG_DIR && PREPEND_JAVA_OPTS="$PREPEND_JAVA_OPTS -verbose:gc -Xloggc:\"$JBOSS_LOG_DIR/gc.log\" -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=3M -XX:-TraceClassUnloading"
```
Comment (danberindei) [inline server/integration/feature-pack/src/main/resources/content/bin/standalone.sh]: I haven't run the test suite yet, but AFAIK `-XX:+PrintGCDateStamps` was removed: 
https://dzone.com/articles/disruptive-changes-to-gc-logging-in-java-9

**Comment by tristantarrant** [comment]
I simply aligned the script with the WF 14 version (plus the Prometheus
changes)

**Comment by tristantarrant** [comment]
CI looks good (although it is JDK 8). I have removed the -XX:+PrintGCDateStamps flag

**Comment by galderz** [comment]
@tristantarrant Should we add JDK 11 CI runs?

**Comment by tristantarrant** [comment]
We already do have nightlies for master

**Pull request merged**